pampa_macros.xml

```diff
@@ -0,0 +1,89 @@
+<macros>
```

Comment:
this file seems to be wrong in this folder

Reply:
oh right! Thanks! Will delete it, but need to be sure this is not the version we need on the right folder...